velox/exec/DistinctAggregations.h

```diff
@@ -24,6 +24,7 @@ namespace facebook::velox::exec {
 
 /// Computes aggregations over de-duplicated inputs. Supports aggregations with
 /// single input column only.
+// TODO consider renaming this class to singular, not plural.
```

Comment:
Let's remove this TODO, but update the sentence before it: "single input column only".

Reply:
done